.gitignore

```diff
@@ -292,3 +292,7 @@ wrapper/
 
 # Pytest artifacts
 tests/*.lock
+
+
+# Minecraft
+minerl/Malmo/Minecraft/run

```

Comment:
Out of curiosity, how is this directory or file generated?

Reply:
Hmm true this is not well documented - it came from the dedicated server

Reply:
Gotcha. Is that the buildkite server?

Reply:
It's a new feature we're working on to use a full minecraft server rather than the integrated one that comes with clients